scripts/rook.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -e
+
+ROOK_VERSION=${ROOK_VERSION:-"v1.1.7"}
+ROOK_DEPLOY_TIMEOUT=${ROOK_DEPLOY_TIMEOUT:-300}
+ROOK_URL="https://raw.githubusercontent.com/rook/rook/${ROOK_VERSION}/cluster/examples/kubernetes/ceph"
+
+function deploy_rook() {
+    kubectl create -f "${ROOK_URL}/common.yaml"
+    kubectl create -f "${ROOK_URL}/operator.yaml"
+    kubectl create -f "${ROOK_URL}/cluster-test.yaml"
+    kubectl create -f "${ROOK_URL}/toolbox.yaml"
+    kubectl create -f "${ROOK_URL}/filesystem-test.yaml"
+    kubectl create -f "${ROOK_URL}/pool-test.yaml"
+
+    for ((retry=0; retry<=ROOK_DEPLOY_TIMEOUT; retry=retry+5)); do
+        echo "Wait for rook deploy... ${retry}s"
+        sleep 5
+
+        if kubectl get cephclusters -n rook-ceph | grep HEALTH_OK &> /dev/null; then
```

Comment:
(nit) is this enough or should we check and wait till the pool and fs are created, just so the automated testing does not rush into testing CSI operations against the same?

Reply:
I had very considered which one is best to know the end of rook-ceph deploy. and I didn't think about that. however i tried with that (rbdpool and fs) it just created after yaml file is run, not ceph deploy. there is no field like status, so I think we can't get result by this.

```shell
whhan91@whhan91-pc:~/go/src/github.com/ceph/ceph-csi$ kubectl get cephfilesystems.ceph.rook.io -A -o wide
NAMESPACE   NAME   ACTIVEMDS   AGE
rook-ceph   myfs   1           7s

whhan91@whhan91-pc:~/go/src/github.com/ceph/ceph-csi$ kubectl get cephblockpools.ceph.rook.io -A -o wide
NAMESPACE   NAME          AGE
rook-ceph   replicapool   15s
```

Reply:
you can check for MDS pod once you  create the filesystem (make sure at least one pod is created) https://github.com/ceph/ceph-csi/blob/master/e2e/deploy-rook.go#L43

in current e2e we are not checking whether the block pool is created or not, with the help of toolbox pod we can check block pool is created or not) this also can be a separate PR

Reply:
I post a new issue to handle this. #730